test/api/groups.coffee

```diff
@@ -151,7 +151,7 @@ describe "Guilds", ->
           done()
 
   context "removing users groups", ->
-    it "allows guild leaders to remove a member", (done) ->
+    it "allows guild leaders to remove a member (but not themselves)", (done) ->
```

Comment:
Technically, each test should be testing one thing, ie: `it "allows guild leaders to remove a member"`, `it "does not allow guild leader to remove themselves"`, `it "does not allow non-guild leaders to remove a member"

Though, that said, we're deprecating these tests soon in favor of more app-agnostic api tests. I'm hoping to set up the new testing suite soon (in ES2015!), so in the meantime, do whatever you want with these